### ISSUE TITLE
"quoted boolean value found" in hash value should not raise a warning.

### ISSUE THREAD

**Issue by sathlan** [rodjek/puppet-lint#474]
Hi,

if in a hash value of them happens to be equal to "true", then this check raise a warning.  Could the hash be excluded from the check ?

``` puppet
my_res {
  ensure => present,
  params => {
      'i_want_this' => 'true',
  }
```

This shouldn't raise any warning.

**Comment by hlindberg** [comment]
Using a string "true" is a perfectly valid thing to do. The check for strings containing "true" was a more widespread problem before facts were structured - then Booleans were in facts strings, and you were more often forced into having checks against "true" where you now use `true`. Because of the old stringified facts, users were led to believe they should use `"true"` elsewhere as well. A general 'warn if string is "true"' option should probably be off by default. if it is 'warn if a directo comparisson or match is made against the string `"true"`, it could be on by default.

Just my 2c.

**Comment by rnelson0** [comment]
See #567 which will fix this. It is targeted for v3.0.0 as disabling it will be a change in user expectations. In the meantime, add this to your Rakefile: `PuppetLint.configuration.send('disable_quoted_booleans')`
